src/odemis/driver/xt_client.py

```diff
@@ -612,6 +624,144 @@ def set_autostigmator(self, channel_name, state):
             self.server._pyroClaimOwnership()
             return self.server.set_autostigmator(channel_name, state)
 
+    def get_pitch(self):
+        """
+        Get the pitch between two beams within the multiprobe pattern.
+
+        Returns
+        -------
+        pitch: float, [um]
+            The distance between two beams of the multiprobe pattern.
+        """
+        with self._proxy_access:
+            self.server._pyroClaimOwnership()
+            return self.server.get_pitch()
+
+    def set_pitch(self, pitch):
+        """
+        Set the pitch between two beams within the multiprobe pattern.
+
+        Returns
+        -------
+        pitch: float, [um]
+            The distance between two beams of the multiprobe pattern.
+        """
+        with self._proxy_access:
+            self.server._pyroClaimOwnership()
+            return self.server.set_pitch(pitch)
+
+    def pitch_info(self):
+        """"Returns the unit and range of the pitch."""
+        with self._proxy_access:
+            self.server._pyroClaimOwnership()
+            return self.server.pitch_info()
+
+    def get_primary_stigmator(self):
+        """
+        Get the control values of the primary stigmator. Within the MBSEM system
+        there are two stigmators to correct for both beamlet astigmatism as well
+        as multi-probe shape. Each stigmator has two control values; x and y.
+
+        Returns
+        -------
+        tuple, (x, y) control values of primary stigmator, unitless.
+        """
+        with self._proxy_access:
+            self.server._pyroClaimOwnership()
+            return self.server.get_primary_stigmator()
+
+    def set_primary_stigmator(self, x, y):
+        """
+        Set the control values of the primary stigmator. Within the MBSEM system
+        there are two stigmators to correct for both beamlet astigmatism as well
+        as multi-probe shape. Each stigmator has two control values; x and y.
+
+        Parameters
+        -------
+        (x, y) control values of primary stigmator, unitless.
+        """
+        with self._proxy_access:
+            self.server._pyroClaimOwnership()
+            return self.server.set_primary_stigmator(x, y)
+
+    def primary_stigmator_info(self):
```

Comment:
For the order of the code I would expect first the info and then the getter/setter.
While reading the code if you wonder what the range is you know that if could be found in the method you just read. But maybe that doesn't make sense to you.

Reply:
It does make more sense, however in the rest of this class the order is first the methods then the info. So for consistency I will leave it like this. If you ever feel like updating the whole class and changing the order go ahead :grin:.